### PR TITLE
Adding 'include' option in configuration

### LINKED ISSUE #142: Include option in couscous.yml
I have a project with many folders in the main repository directory, but only one directory for the Markdown documentation, so `exclude`-ing every single folder seems like a waste. I think it would be value added if there was an `include` option that will *only* include the folder(s) you specify.

### PR DESCRIPTION
**Pull request by wysow**
Done this really quickly but does the job, what do you think guys?

Resolve #142

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -18,6 +18,11 @@ template:
     # Name of the index file (default is "README.md")
     index: index.md
 
+# List of directories to include in the processing (default is all markdown files)
+# Paths are relative to the repository root
```
Comment (mnapoli) [inline docs/configuration.md]: Could you maybe make it explicit that it's empty by default (if empty then it's "all markdown files")

**Comment by mnapoli** [comment]
Looks good to me, could you add a [functional test](https://github.com/CouscousPHP/Couscous/tree/master/tests/FunctionalTest/Fixture)?

**Comment by wysow** [comment]
@mnapoli and here are some tests... ;-)

**Comment by mnapoli** [comment]
Awesome thanks!

**Pull request merged**